### ISSUE TITLE
Inconsistency between activity text and activity indicator in title.

### ISSUE THREAD

**Issue by jainkuniya** [zulip/zulip-mobile#3147]
<img width="352" alt="image" src="https://user-images.githubusercontent.com/18511177/48637878-f9615e00-e9f4-11e8-8d5b-82d936047299.png">

In the above image, activity text is "Active Now", but indicator color is not green. It should be green.

@zulipbot claim

**Comment by borisyankov** [comment]
I suspect the color indicator is the incorrect one 😄

**Comment by borisyankov** [comment]
We might want to get rid of the intermediate yellow state altogether and be either no indicator or a green dot.

**Comment by gnprice** [comment]
Thanks @jainkuniya for flagging this!

In the webapp, if I
* go through the people in the right sidebar with orange "idle" half-dots,
* click on one of the chevrons to get a user's profile,
* and hover on the orange half-dot within the profile,
* the hover info consistently says "Just now".

I suspect the behavior reported here is what we're doing in the exact same situation. In the presence system's terms:
https://zulip.readthedocs.io/en/latest/subsystems/presence.html
I think what's happening is that the user has a client which is online now... but they haven't actually interacted with the client recently.

I'd say the combination of an orange "idle/inactive" indicator (AFAICT we don't have any words within the app to explain what it means) with the text "Just now" is pretty confusing in itself, and probably isn't a fully-considered product choice.

So, we should figure out how we want this system to behave as a product matter, and describe that clearly. Then the mobile app and webapp can both implement the behavior we decide we want.

@rishig , do you have a sense of how you'd like this area to work? /cc also @timabbott who may have more to add on how the underlying system is working. It looks like zulip/zulip#11607 (an ongoing open issue) is related, too.

**Comment by rishig** [comment]
yeah, I think https://github.com/zulip/zulip/issues/11607 has my updated thoughts. Agreed that "last seen" should have something like "N minutes ago" if they are not green; not sure if the current webapp behavior is a bug or a previous product decision.

**Comment by timabbott** [comment]
@rishig can you collaborate with @vinitS101 on getting the design for https://github.com/zulip/zulip/pull/11882 (the PR for https://github.com/zulip/zulip/issues/11607) to something you're happy with?  I think that's probably what's blocked there.  

The fundamental problem here is that "orange" is a weird state to explain, and we don't explain it.  The person has a connected client that's online now, but we don't know if they're at their computer.  We have plans involving the desktop app to know for sure whether the user is at their computer (and thus be able to turn a lot of orange -> green), but we'll likely never be able to do that in the webapp.  Maybe "Online, last active 12 minutes ago" might be a clean way to summarize orange states.
